pimd/pim_rp.c

```diff
@@ -968,8 +958,9 @@ void pim_rp_check_on_if_add(struct pim_interface *pim_ifp)
 			rp_info->i_am_rp = 1;
 			if (PIM_DEBUG_PIM_NHT_RP) {
 				char rp[PREFIX_STRLEN];
-				pim_addr_dump("<rp?>", &rp_info->rp.rpf_addr,
-					      rp, sizeof(rp));
+
+				snprintfrr(rp, sizeof(rp), "%pPA",
```

Comment:
Useless `snprintfrr`, use `%pPA` in zlog_debug directly

Reply:
Fixed